### DISCUSSION TITLE
Annotation Editor Layer doesnt rendering in correct sizes

### DISCUSSION THREAD

**Discussion by ademyalcin27** [mozilla/pdf.js#18142] [Q&A] [answered]
Attach (recommended) or Link to PDF file here:
When I wanted to use the annotation editor layer I realized it doesn't render the correct size of the page. you will see extra space right and bottom its not covering page 100% it looks bigger than page.

Configuration:

Web browser and its version: chrome Version 125.0.6422.61 (Official Build) (x86_64)
Operating system and its version: Mac OS
PDF.js version: 4.2.67
Is a browser extension:
Steps to reproduce the problem:
1. https://stackblitz.com/edit/nuxt-starter-xdqtqv
2.

What is the expected behavior? (add screenshot)
![image](https://github.com/mozilla/pdf.js/assets/5026172/12a65465-4715-4c09-ba2c-3314430554d4)

What went wrong? (add screenshot)
![image](https://github.com/mozilla/pdf.js/assets/5026172/e9348da4-705d-46ed-806c-211b03f3d280)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by ademyalcin27** [comment]
@Snuffleupagus why do you converting to discussion? I added whole information that you need it

**Comment by Snuffleupagus** [comment]
> why do you converting to discussion? I added whole information that you need it

Because this unfortunately appears to be an issue in your application, rather than in the PDF.js library itself.
Also, your example is large/complex enough that it's probably not trivial for anyone to provide assistance.

**Comment by Zixtar** [comment]
I've had a similar issue in my app, I've fixed it mostly by removing css borders . For example in my case a substantial change can be seen by removing the page border:
![image](https://github.com/mozilla/pdf.js/assets/24939052/c9e377f1-e49c-4d22-98ba-15585a8fa220)

**Reply by ademyalcin27** [reply]
@Zixtar  thank you for suggestion
I will try it this or I have also another idea when an annotation is rendered I will reset the width and height. of course this is not good solution but if there is better solution Im open the suggestions

**Comment by nicolo-ribaudo** [comment, marked as answer]
@ademyalcin27 The problem is that you are setting `box-sizing: border-box` on the `.page` div

**Reply by ademyalcin27** [reply]
this solved my whole problems. thank you @nicolo-ribaudo  :pray:
